test/testcafe/spec/EmailMagicLinkOTPTerminalView_spec.js

```diff
@@ -79,3 +99,23 @@ async function setupOtpOnly(t) {
       }
     });
 });
+
+// Make sure form title, otp-only warning have updated wording, user email and mobile icon are present in the response
+[
+  ['Should have entry for enter code on sign in page and mobile icon (Android) when accessed', terminalReturnOtpOnlyFullLocationMobileIconAuthenticationMock, 'Enter this code on the sign in page.'],
+  ['Should have entry for enter code on password reset page and mobile icon (Android) when accessed', terminalReturnOtpOnlyFullLocationMobileIconRecoveryMock, 'Enter this code on the password reset page.'],
+  ['Should have entry for enter code on account unlock page and mobile icon (iOS) when accessed', terminalReturnOtpOnlyFullLocationMobileIconUnlockMock, 'Enter this code on the account unlock page.'],
+  ['Should have entry for enter code on registration page and mobile icon (iOS) when accessed', terminalReturnOtpOnlyFullLocationMobileIconEnrollmentMock, 'Enter this code on the sign up page.'],
```

Comment:
Tests looks sweet, nice usage of parameterized tests 🔥

Reply:
I learn from the best :D